docs/INSTALLATION.md

```diff
@@ -78,7 +78,15 @@ means that the open terminals don't know that a new program was installed.
 >
 > Close any open terminals and open a new one.
 
-# Assignment Requirements
+### Assignment Requirements
```

Comment:
Single `#` does not render properly:

<img width="755" alt="Screenshot 2020-03-14 at 5 21 24 PM" src="https://user-images.githubusercontent.com/980783/76681411-6819f000-6618-11ea-925b-ede96e4ebffb.png">